pkg/selinux/selinux-client.js

```diff
@@ -101,8 +103,24 @@ client.init = function(statusChangedCallback) {
         );
     };
 
-    refreshInfo();
-    window.setInterval(refreshInfo, pollingInterval);
+    var polling = null;
+
+    function setupPolling() {
+        if (cockpit.hidden) {
+            window.clearInterval(polling);
+            polling = null;
+        } else if (polling === null) {
+            polling = window.setInterval(refreshInfo, pollingInterval);
+            refreshInfo();
+        }
+    }
+
+    cockpit.addEventListener("visibilitychange", setupPolling);
+    setupPolling();
+
+    /* The first time */
+    if (polling === null)
```

Comment:
This should be covered by the call to `setupPolling` above. If the frame is hidden, it is unlikely we need to gather the info I would say.